Apps/Examples/Examples/All Examples/TrackingModeExample.swift

```diff
@@ -32,6 +40,35 @@ public class TrackingModeExample: UIViewController, ExampleProtocol {
             self.finish() // Needed for internal testing purposes.
         }
     }
+
+    @objc func showHideBearingImage() {
+        showsBearingImage.toggle()
+    }
+    func syncPuckAndButton() {
+        // Update puck config
+        let configuration = Puck2DConfiguration.createDefault2DPuck(withBearing: showsBearingImage)
+
+        mapView.location.options.puckType = .puck2D(configuration)
+
+        // Update button title
+        let title: String = showsBearingImage ? "Hide bearing image" : "Show bearing image"
+        toggleBearingImageButton.setTitle(title, for: .normal)
+    }
+    private func setupToggleShowBearingImageButton() {
+        // Styling
+        toggleBearingImageButton.backgroundColor = .systemBlue
+        toggleBearingImageButton.addTarget(self, action: #selector(showHideBearingImage), for: .touchUpInside)
+        toggleBearingImageButton.setTitleColor(.white, for: .normal)
+        toggleBearingImageButton.isHidden = false
+        syncPuckAndButton()
+        toggleBearingImageButton.translatesAutoresizingMaskIntoConstraints = false
+        view.addSubview(toggleBearingImageButton)
+
+        // Constraints
+        toggleBearingImageButton.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 20.0).isActive = true
+        toggleBearingImageButton.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -20.0).isActive = true
+        toggleBearingImageButton.topAnchor.constraint(equalTo: view.topAnchor, constant: 650.0).isActive = true
```

Comment:
Looks like you're trying to place the button at the bottom of the screen? Different phones have different screen heights, so use `toggleBearingImageButton.bottomAnchor` and `view.safeAreaLayoutGuide.bottomAnchor` (anchoring to the safe area helps the layout work with iPhones that don't have a home button).